easybuild/tools/github.py

```diff
@@ -617,9 +619,14 @@ def _easyconfigs_pr_common(paths, start_branch=None, pr_branch=None, target_acco
 
     _log.debug("git status: %s", git_repo.git.status())
 
-    # copy files to right place
+    deps = resolve_dependencies([process_easyconfig(ec)[0] for ec in paths], modules_tool())
+
     file_info = copy_easyconfigs(paths, os.path.join(git_working_dir, pr_target_repo))
 
+    # dont save info on dependencies
+    dep_paths = [dep['spec'] for dep in deps if dep['spec'] not in paths]
```

Comment:
are the `paths` specified as absolute paths here? I'd expect not, so the `not in` check isn't going to do what is intended...